search/search_repository.go

```diff
@@ -373,6 +373,8 @@ func (r *GormSearchRepository) SearchFullText(ctx context.Context, rawSearchStri
 func init() {
 	// While registering URLs do not include protocol becasue it will be removed before scanning starts
 	// Please do not include trailing slashes becasue it will be removed before scanning starts
-	RegisterAsKnownURL("work-item-details", `(?P<domain>demo.almighty.io)(?P<path>/work-item-list/detail/)(?P<id>\d*)`)
-	RegisterAsKnownURL("localhost-work-item-details", `(?P<domain>localhost)(?P<port>:\d+){0,1}(?P<path>/work-item-list/detail/)(?P<id>\d*)`)
+	RegisterAsKnownURL("work-item-details", `(?P<domain>demo.almighty.io)(?P<path>/work-item/list/detail/)(?P<id>\d*)`)
```

Comment:
@aslakknutsen 
```
hostString := configuration.GetHTTPAddress()
urlRegexString := fmt.Sprintf("(?P<domain>%s)(?P<path>/work-item-list/detail/)(?P<id>\\d*)", hostString)
search.RegisterAsKnownURL("work-item-details", urlRegexString)
```
Doing this as a setup in `main.go` makes it possible to have known URLs registered while system start.